<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>911</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="page">
     <property name="styleSheet">
      <string notr="true">#page{
	
	
	background-image: url(:/new/prefix1/ouv_ct_fotolia.jpg);
Background-color : Lightgray;
border : 3px solid gray;
border-radius : 40px;}</string>
     </property>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>331</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#frame{
border : 3px solid black;
border-radius : 40px;}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 85, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 85, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>date debut</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>10</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 85, 255);
font: 13pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Ajout</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 85, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="id">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>90</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 85, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>date fin</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ajout">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>280</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 85, 255);</string>
       </property>
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>280</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 85, 255);</string>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 85, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Local</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="local">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>130</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 85, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>nombre personne</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nbr_pr">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>170</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>210</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>250</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>50</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>civil</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>administrative</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>criminel</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>40</y>
        <width>431</width>
        <height>451</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{background:transparent} QTabWidget::pane{border: 3px;border-color:solid black ;background-color: transparent;} 
QTabBar::tab {background-color: transparent;color: black;width: 100px;height:20px;font-size:24px} 
QTabBar::tab:hover{background-color:  rgb(255, 151, 6); color: black;}
QTabBar::tab:selected{background-color:  rgb(170, 85, 255); color: black;}
</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QFrame" name="frame2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>411</width>
          <height>391</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#frame2{
border : 3px solid black;
border-radius : 40px;}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>0</y>
           <width>151</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 85, 255);
font: 13pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Calendrier</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QCalendarWidget" name="calendarWidgett">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>351</width>
           <height>218</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(238, 51, 255);</string>
         </property>
        </widget>
        <widget class="QDateEdit" name="dateEdit_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>250</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_6">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>290</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>290</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>320</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 170, 0);</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>350</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>320</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_7">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>350</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>280</y>
           <width>51</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 85, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>280</y>
           <width>31</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 85, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>310</y>
           <width>51</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 85, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Local</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>310</y>
           <width>71</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 85, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>nbr per</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>340</y>
           <width>71</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 85, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>date_D</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_17">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>340</y>
           <width>61</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 85, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>date_F</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="QFrame" name="frame_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>401</width>
          <height>331</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#frame_2{
border : 3px solid black;
border-radius : 40px;}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="pushButton_5">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>20</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 85, 255);</string>
         </property>
         <property name="text">
          <string>Extraire</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>10</y>
           <width>41</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 85, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>PDF</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_5">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>50</y>
           <width>211</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QTableView" name="tableView">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>100</y>
           <width>361</width>
           <height>171</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="supp">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>280</y>
           <width>81</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 85, 255);</string>
         </property>
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
        <widget class="QPushButton" name="trie">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>280</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 85, 255);</string>
         </property>
         <property name="text">
          <string>trier</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>280</y>
           <width>81</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>ID</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DATE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TYPE</string>
          </property>
         </item>
        </widget>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>21</y>
           <width>151</width>
           <height>16</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_6">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>50</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 85, 255);</string>
         </property>
         <property name="text">
          <string>Extraire</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>49</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="smart.qrc">:/new/prefix1/4599811.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>40</y>
           <width>41</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 85, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Excel</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>280</y>
           <width>81</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 85, 255);</string>
         </property>
         <property name="text">
          <string>charger</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="stat">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <widget class="QFrame" name="frame_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>411</width>
          <height>411</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#frame_3{
border : 3px solid black;
border-radius : 40px;}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="widget" native="true">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>341</width>
           <height>281</height>
          </rect>
         </property>
        </widget>
        <widget class="QFrame" name="frame_4">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>320</y>
           <width>31</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 0, 0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
        <widget class="QFrame" name="frame_5">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>350</y>
           <width>31</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 255, 0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
        <widget class="QFrame" name="frame_6">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>380</y>
           <width>31</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(98, 213, 255);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_18">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>380</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>civil</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_19">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>350</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>criminel</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_20">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>320</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>administrative</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1026</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="smart.qrc"/>
 </resources>
 <connections/>
</ui>
